src/sum_of_nth.py

```diff
@@ -0,0 +1,11 @@
+"""Codewars Sum of the first nth term of series."""
+
+
+def series_sum(n):
+    """Return the nth term of the series: 1 + 1/4 + 1/7 + 1/10 + 1/13."""
+    sum = 0
```

Comment:
You shouldn't be using the `sum` keyword here as a variable. It's a Python built-in